index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/fontawesome.min.css">
+    <title>Francis</title>
+</head>
+<body>
+    <main id="main-container"   >
+        <!-- Top navigation section -->
+        <div id="nav-headline-background">
+            <nav id="top-navigation">
+                <div id="logo">
+                    <p id="logo-text">Welcome</p>
+                </div>
+                <ul id="navigation-list">
+                    <li class="navigation-item"><a href="index.html" class="navigation-item-text">Portfolio</a></li>
+                    <li class="navigation-item"><a href="#about" class="navigation-item-text">About</a></li>
+                    <li class="navigation-item"><a href="#form" class="navigation-item-text">Contact</a></li>
+                </ul>
+                <div id="toolbar-menu-button">
+                    <div id="toolbar-menu-icon">
+                        <div id="toolbar-menu-union-icon">
+                        </div>
+                    </div>
+                </div>
+        
+            </nav>
+
+            <!-- Headline section -->
+            <section id="headline">
+                <h1 class="headline-heading heading-text">Hey there. I'm Francis.
+                    <span id="color-this">I'm a software developer</span>
+                </h1>
+
+                <p class="headline-description description-text">I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+
+                <div id="headline-social-media">
+                    <div class="social-media">
+                        <section class="social-media-icon"><a href="http//:github.com/francis450"><img src="img/social-media-icons/github.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:linkedin/in/njoroge-francis"><img src="img/social-media-icons/linkedin.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:evangelist"><img src="img/social-media-icons/angelist.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:twitter.com/nkamandefrancis"><img src="img/social-media-icons/twitter.svg"></img></a></section>
+                        <section class="social-media-icon"><a href="http//:medium.com"><img src="img/social-media-icons/medium.svg"></img></a></section>
+                    </div>
+                </div>
```

Comment:
- [ ] I would suggest using ```ul``` with ```li``` tags for this section instead of divs since this is more semantically sound. Great work however on the use of anchor tags of the social media icons 👍